src/evotorch/core.py

```diff
@@ -3762,6 +3762,38 @@ def __iter__(self):
         for i in range(len(self)):
             yield self[i]
 
+    def to(self, device: Device) -> "SolutionBatch":
```

Comment:
It would be cool if this also supported to dtype? I'm not sure how much extra work it would be but could be a nice extra feature

Reply:
Nice idea, thanks. For now, could you please make an issue so that we do not forget this idea? We can implement this over time.

Reply:
@NaturalGradient open issue please